packages/aria2.rb

```diff
@@ -28,13 +28,6 @@ class Aria2 < Package
   depends_on 'libxml2'
   depends_on 'sqlite'
   depends_on 'zlibpkg'
```

Comment:
Are c_ares, libssh2, libxml2, sqlite and zlibpkg core packages?  If so, no need to list here.

Reply:
Core packages are defined by anything build_essential pulls in or are they the packages pullied in by install.sh before crew runs?

In any case, which core package list should we check this against?

Reply:
https://github.com/skycocker/chromebrew/blob/master/tools/core_packages.txt

Reply:
Thanks for that. Taken care of.